crates/xtask/src/main.rs

```diff
@@ -11,34 +11,29 @@ mod scratch;
 mod stage;
 mod test;
 
-#[derive(Parser)]
-#[command(author, version, about, long_about = None)]
-struct Cli {
-    #[command(subcommand)]
-    command: Commands,
+#[macro_export]
+macro_rules! make_cli {
+    ($first:expr, $($segments:expr),+) => {};
 }
 
-#[derive(Subcommand)]
-enum Commands {
-    Scratch,
-    Dist,
-    Install { game_dir: String },
-    Test,
-}
+make_cli!(dist, install, scratch);
```

Comment:
what is this doing? :thinking: the macro is now empty

Reply:
No idea what you're talking about. You saw nothing. I certainly didn't leave a bit of dead code from a path I decided not to take. 😅